docs/wiki/EL/el-architecture.md

```diff
@@ -173,12 +173,39 @@ At its core, Geth runs the Ethereum Virtual Machine (EVM), which processes smart
 
 🔗 [EVM Code in Geth]( https://github.com/ethereum/go-ethereum/blob/master/core/vm/evm.go#L90)
 
-##### Sync
+##### State Sync
 
-Execution client synchronizes the chain by downloading block data from its peers and verifying them using the block validation rule. Sync finishes when blockchain data is verified and clients catches up with the tip of the chain which enables building the latest state. 
+All execution clients require an up‑to-date world state to validate and build blocks. To bootstrap a new node's current state, clients leverage Ethereum’s DevP2P subprotocols: `eth/*` (wire protocol) for block headers, bodies, and receipts, and `snap/1` for creating state snapshots. Using these sub-protocols, clients can choose between two sync strategies, **full sync** or **snap sync**.  The difference between the snap synced node and a full block-by-block synced node is that a snap synced node started from an initial checkpoint that was more recent than the genesis block.
 
-Because it's inefficient to validate block by block and transaction by transaction since the genesis, EL clients employ other strategies to securely sync the tip of the chain, e.g. [snap sync](https://github.com/ethereum/devp2p/blob/master/caps/snap.md?plain=1#L1)
+Let's look at how the flows of both **full sync** and **snap sync** work.
 
+### Full Sync
+Full Sync trades absolute trustlessness for time and resources. You replay every block and transaction from genesis to tip, rebuilding the state trie step by step:
```

Comment:
```suggestion
Full Sync trades absolute trustlessness for time and resources. The client replays every block and transaction from genesis to tip, rebuilding the state trie step by step:
```

Reply:
I would maybe add that full sync is the default naive strategy and because it's starting from genesis, it's not sustainable for the future (4444)

Reply:
solid point! added 4444 context in latest commit https://github.com/eth-protocol-fellows/protocol-studies/pull/437/commits/2bde312198c716340ca067a292936d77415c1ae6

Other thoughts I didn't include in the PR:
On top of 4444, once we get statelessness, syncing should be significantly sped up, since each block will have execution witnesses, containing each account and storage slots accessed, and block validation can just run txs against an in-memory state trie created from those accessed accounts and storage slots.
^^ maybe too early to add this context to the sync wiki section but def going to be a game changer